src/odemis/acq/stream/test/stream_sparc2_streakcam_test.py

```
@@ -0,0 +1,830 @@
+
+        streaks.image.subscribe(self._on_image)
+
+        # shouldn't affect
+        streaks.roi.value = (0.15, 0.6, 0.8, 0.8)
+        streaks.repetition.value = (5, 6)
+
+        # set GUI VAs
+        streaks.detExposureTime.value = 0.5  # s
+        streaks.detBinning.value = (2, 2)  # TODO check with real HW
+        streaks.detStreakMode.value = True
+        streaks.detTimeRange.value = find_closest(0.000000005, self.streak_unit.timeRange.choices)
```

Ruff (0.15.12)

[error] Undefined name `find_closest`

(F821)